management/backup.py

```diff
@@ -457,6 +457,13 @@ def list_target_files(config):
 
 		return [(key.name[len(path):], key.size) for key in bucket.list(prefix=path)]
 	elif target.scheme == 'b2':
+		# InvalidBackendURL error for B2 backend if application key contains a '/' character
+		# See: https://bugs.launchpad.net/duplicity/+bug/1819390
+		# With a slash anywhere after b2::// the above urlparse will put something into target.path, thus
+		if not "".__eq__(target.path):
```

Comment:
Why didn't you write `target.path != ""`?

Reply:
something I saw on stackexchange - I'll test with plainer version